test/helpers/compile_deploy.js

```diff
@@ -0,0 +1,50 @@
+const { readFileSync } = require("fs");
+const { compile } = require("solc");
+
+/**
+ * Compile and deploy the selected contract(s)
+ * @param {String} contractPath  Path to contracts directory
+ * @param {String} mainContractName  Name of the main contract (without .sol extension)
+ * @param {Array|String} contractFileNames List of imported contracts
+ * @param {Object} web3 Web3 interface
+ * @returns {Object} context
+ */
+async function compileAndDeploy(contractPath, mainContractName, contractFileNames = [], web3) {
```

Comment:
doesn't this duplicate `test/helpers/contracts.js` ?

Reply:
It adds multi-contract support and also export accounts. Also some refactoring.